cloud-sql-proxy/README.MD

```diff
@@ -64,6 +64,80 @@ gcloud dataproc clusters create <CLUSTER_NAME> \
     --metadata "additional-cloud-sql-instances=<PROJECT_ID>:<REGION>:<ANOTHER_INSTANCE_NAME>"
 ```
 
+## Private IP Clusters and Cloud SQL Instances
+
+Connecting to a cloud sql instance from a private IP cluster requires some additional setup.
+
+1. Create a network.
+
+First, create a network:
+```
+gcloud compute networks create <NETWORK>
+```
+Note that regional subnets will be created automatically.
+To list these subnets:
+```
+gcloud compute networks subnets list --network <NETWORK>
+```
+
+2. Enable private IP for a subnet.
+
+Choose the subnet that is in the region you want to use.
+Enable private ip google access for your subnet:
+```
+gcloud compute networks subnets update <SUBNET> \
+    --region <REGION> \
+    --enable-private-ip-google-access
+```
+
+3. Create a cloud sql instance
+
+Create a second generation cloud sql instance:
+
+```
+gcloud sql instances create <INSTANCE_NAME> \
+    --database-version="MYSQL_5_7" \
+    --activation-policy=ALWAYS \
+    --gce-zone <ZONE>
```

Comment:
WARNING: Flag `--gce-zone` is deprecated and will be removed by release 255.0.0. Use `--zone` instead.

Reply:
Done